### ISSUE TITLE
Internal Memory Error, Formatting... PRS600,  (650, 950)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [natowi/prs-plus#366]
```
Your model PRS 600 / PRS+ 2.1.01a

What steps will reproduce the problem?
1.Transfer ebooks from calibre, disconnect reader
1.1 or open ebook file
2. The problem seems to appear irregular
3. The following message box appears:

PRS600 STRING
(de) <string id="STR_UI_CONFIRM_INVALIDFORMAT" value="Ungewöhnlicher 
Gerätespeicherzustand&#xA;Formatiert." />
(en)  <string id="STR_UI_CONFIRM_INVALIDFORMAT" value="Exceptional internal 
memory state&#xA;Formatting." />

May also be relevant for PRS 650 and PRS 950
(en)  <string id="STR_UI_CONFIRM_INVALIDFORMAT" value="Exceptional internal 
memory state&#xA;Formatting." />

I found the messages in .../resources/scripts/languages/en/..

It is only possible to click on --<OK>--

Unfortunately my Solution to this problem is -not- reliable:
-connect reader with computer (without running Calibre or Sony Library), 
-backup files and disconnect reader.
The message will disappear or after formatting memory you can copy the files 
back.

Sample location of the Bug files:

READER:\database\media\books\Name, Test

F-8"?>
<.del      Change date: 20.03.2038 Size: 1.957.324KB
ice/1"/>.
        0KB

These files cause the INTERNAL MEMORY ERROR!
Files can not be edited or removed.
(Only after formatting internal memory)

If there is no idea, how to fix this "Exceptional State"-problem, changing the 
message box from 

"Exceptional internal memory state&#xA;Formatting"
--<OK>--

to something like

"Exceptional internal memory state&#xA;Please connect Reader 
with your Computer to backup it. Reader will format the internal memory!"
--<Switch off>-- or --<Format>--

would be helpful.

Otherwise an user, who did not expect this "Exceptional State" massage will 
click --<OK>-- 
and loose his files.
After the modification he will be able to backup his files and switch the 
reader off (when no computer is available).

I would be pleased about a fix, but I know it has low priority.
Thanks




```

Original issue reported on code.google.com by `simf...@googlemail.com` on 24 Jan 2013 at 4:19

Attachments:
- [error.JPG](https://storage.googleapis.com/google-code-attachments/prs-plus/issue-366/comment-0/error.JPG)

**Comment by gingerbeardman** [comment]
I just got this. I lost all my custom fonts, which are not backed up by calibre.

**Comment by gingerbeardman** [comment]
Just had this again, but it in the intervening years I had figured out a workaround for the "Exceptional internal memory state. Formatting."

1. Plug device into the computer
1. Backup READER partition
1. Press RESET button
1. Wait for Reader to reboot (no PRS+ features when booting with USB connected)
1. Unplug from computer
1. Shutdown/restart reader (PRS+ features will be re-enabled)
1. Problem is gone!

My theory is that PRS+ doesn't the device being "dirty" when it is unmounted or ejected improperly.
